tiledb/sm/array_schema/dimension.cc

```diff
@@ -636,6 +655,60 @@ void Dimension::set_oob_func() {
   }
 }
 
+void Dimension::set_value_in_range_func() {
+  // Set
```

Comment:
Incomplete comment here?

Reply:
Good catch, I am removing in my current batch and the change will appear in my subsequent PR.